ambari-server/src/test/java/org/apache/ambari/server/agent/HeartbeatProcessorTest.java

```diff
@@ -433,6 +434,8 @@ public void testHeartbeatCustomStartStop() throws Exception {
   }
 
   @Test
+  @Ignore
+  //TODO should be rewritten, componentStatuses already are not actual as a part of heartbeat.
```

Comment:
Why not delete these tests, do we not have test coverage for status sent from agent?